src/main/oAuthClient.ts

```diff
@@ -49,6 +49,13 @@ export type LogoutParams = {
   removeCredentials?: boolean
 }
 
+export type RevocationParams = {
+  token: string
+  tokenTypeHint: TokenTypeHint
```

Comment:
TBH, I don't think the hint is something we should require the SDK user to input. Instead, I'd pass either:
- an object `{ accessToken?, refreshToken? }`, we make the calls ourselves and indicate the hint.
- two params `accessToken?`, `refreshToken?`, we make the calls ourselves and indicate the hint.
- or just an array of tokens `tokens?: string[]` ; the hint is optional anyway 🤷 
-